examples/animation/pause_resume.py

```diff
@@ -0,0 +1,46 @@
+"""
+==================================
+Pausing and Resuming and Animation
+==================================
+
+This example showscases:
+- using the Animation.pause() method to pause an animation.
+- using the Animation.resume() method to resume an animation.
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import numpy as np
+
+
+class PauseAnimation:
+    def __init__(self):
+        fig, ax = plt.subplots()
+        x = np.linspace(-0.1, 0.1, 1000)
```

Comment:
Can you add a title to the figure indicating that the user should click to pause/resume?

Reply:
Done.